src/ensembl/src/shared/accordion/Accordion/Accordion.test.tsx

```diff
@@ -0,0 +1,222 @@
+          .length
+      ).toEqual(1);
+    });
+
+    it('collapses an expanded item when its title is clicked', () => {
+      const wrapper = mountAccordion();
+
+      const fooTitle = wrapper.find(FooTitle);
+
+      fooTitle.simulate('click'); // open
+      fooTitle.simulate('click'); // close
+
+      const instance = wrapper.find(Provider).instance() as Provider;
+
+      expect(
+        instance.state.items.filter((item: Item) => item.expanded === true)
+          .length
+      ).toEqual(0);
+    });
+  });
+
+  describe('<Accordion accordion="false" /> (‘tabpanel’)', () => {
+    const [FooTitle, BarTitle] = [
+      (): JSX.Element => <AccordionItemTitle>Foo</AccordionItemTitle>,
+      (): JSX.Element => <AccordionItemTitle>Bar</AccordionItemTitle>
+    ];
+
+    function mountTabpanel(): ReactWrapper {
+      return mount(
+        <Accordion accordion={false}>
+          <AccordionItem>
+            <FooTitle />
+          </AccordionItem>
+          <AccordionItem>
+            <BarTitle />
+          </AccordionItem>
+        </Accordion>
+      );
+    }
+
+    it('expands a collapsed item when its title is clicked', () => {
+      const wrapper = mountTabpanel();
+
+      wrapper.find(FooTitle).simulate('click');
+
+      const instance = wrapper.find(Provider).instance() as Provider;
+
+      expect(
+        instance.state.items.filter((item: Item) => item.expanded === true)
+          .length
+      ).toEqual(1);
+    });
+
+    it("expands a collapsed item when its title is clicked, and doesn't close the others", () => {
+      const wrapper = mountTabpanel();
+
+      wrapper.find(FooTitle).simulate('click');
+      wrapper.find(BarTitle).simulate('click');
+
+      const instance = wrapper.find(Provider).instance() as Provider;
+
+      expect(
+        instance.state.items.filter((item: Item) => item.expanded === true)
+          .length
+      ).toEqual(2);
+    });
+
+    it('collapses an expanded item when its title is clicked', () => {
+      const wrapper = mountTabpanel();
+
+      wrapper.find(FooTitle).simulate('click'); // open
+      wrapper.find(FooTitle).simulate('click'); // close
+
+      const instance = wrapper.find(Provider).instance() as Provider;
+
+      expect(
+        instance.state.items.filter((item: Item) => item.expanded === true)
+          .length
+      ).toEqual(0);
+    });
+  });
+
+  it('does not expanded disabled items on click', () => {
+    const wrapper = mount(
+      <Accordion accordion={false}>
+        <AccordionItem disabled={true}>
+          <AccordionItemTitle className="foo">Foo Title</AccordionItemTitle>
+        </AccordionItem>
+      </Accordion>
+    );
+
+    const instance = wrapper.find(Provider).instance() as Provider;
+
+    wrapper
+      .find('.foo')
+      .first()
+      .simulate('click');
+    expect(
+      instance.state.items.filter((item: Item) => item.expanded === true).length
+    ).toEqual(0);
+  });
+
+  it('pre expanded accordion', () => {
```

Comment:
Is this about an item being expanded by default? If yes, how about `it('expands by default')`?

Reply:
I've updated it to `pre expands the item that has expanded === true`. That sounds okay?

Reply:
`pre-expands the item that is marked as expanded` ?

(to avoid the implementation detail of `expanded=true`, and to avoid detachment of the prefix from the word it belongs to 😄 )